src/components/Listings/index.js

```diff
@@ -18,7 +18,7 @@ function parseName(name) {
     return "Anonymous"
   }
 
-  const [first, ...rest] = name.trim().split(" ")
+  const [first, ...rest] = name.trim().split("/\s+/")
```

Comment:
@mewtaylor nice catch! We should do this scrubbing on the public sheet rather than in JS for privacy reasons though.

Reply:
Great point, thanks for catching that!

Reply:
https://trello.com/c/u8YQ13eW